### PR TITLE
Feature/issue 2 added historical price endpoint

### PR DESCRIPTION
**Pull request by animuku**
Link to document - https://docs.google.com/document/d/1fdPUKK6ey2y9UGnt0gFwGQfzolSufPulfS_e1pzcZtc/edit?usp=sharing

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+iexToken=${IEX_TOKEN}
```
Comment (grundygolem2) [inline src/main/resources/application.properties]: try to move this into the yml to avoid multiple config files

**Review by grundygolem2 — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (animuku) [reply src/main/resources/application.properties:1]: I tried adding it to the YML first, but for some reason it doesn't read the environment variable. It works if I add it to application.properties, so I created that file.

Comment (animuku) [reply src/main/resources/application.properties]: For some reason, it wasn't working before when I put it inside the YML. So, I added it to application.properties. But I re-checked now and it works inside the YML as well, so I'll make that change.

Comment (animuku) [reply src/main/resources/application.properties]: I've made the changes.

**Review by grundygolem2 — APPROVED** [review]

**Pull request merged**